client/constants.py

```diff
@@ -23,7 +24,8 @@ class StatusCode(object):
                     UNAUTHORIZED: "Unauthorized",
                     RATE_LIMIT: "Too Many Requests",
                     FORBIDDEN: "Forbidden",
-                    WEBSITE_REQUEST_ERROR: "Website Request Error"}
+                    WEBSITE_REQUEST_ERROR: "Website Request Error",
```

Comment:
I feel like this error doesn't explain what is actually the problem. According to https://www.smartlabsoftware.com/ref/http-status-codes.htm#code-550
It says that 550 is a Permission Denied error. But http://stackoverflow.com/questions/16525799/proper-use-of-http-error-codes-550-why-is-it-5xx-instead-of-4xx states that 550 isn't a recognized error code. So can we get some better naming/explanation around this?